reconstruction/MRI_reconstruction/varnet_demo/inference.ipynb

```diff
@@ -19,6 +31,7 @@
   },
   {
    "cell_type": "markdown",
+   "id": "f85aee82",
    "metadata": {},
    "source": [
     "# Import packages"
```

Comment:
I think this notebook lacks the standard cells for "Setup environments" and "Setup imports"